packages/sdk/contractkit/src/wrappers/Election.test.ts

```diff
@@ -19,6 +19,8 @@ const blsPublicKey =
 const blsPoP =
   '0xcdb77255037eb68897cd487fdd85388cbda448f617f874449d4b11588b0b7ad8ddc20d9bb450b513bb35664ea3923900'
 
+jest.setTimeout(20000)
```

Comment:
this should fix the test flakiness (does so locally at least), looks like this is caused by a timeout on the findLesserGreater tests